### PR TITLE
Remove transaction logging

### PR DESCRIPTION
**Pull request by abizjak**
## Purpose

Part of https://github.com/Concordium/concordium-client/pull/160

## Changes

Remove the postgres and database specific parts from the base. This removes the postgres dependency from base.
These have been moved to the wallet-proxy where they are needed.

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [x] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

File: haskell-src/Concordium/ID/Types.hs [hunk 1]
```diff
@@ -1058,7 +1058,7 @@ values (NormalACWP cdi) = NormalAC (cdiValues cdi) <$> proofCommitments (cdiProo
 class HasCredentialType a where
   credentialType :: a -> CredentialType
 
-instance HasCredentialType AccountCredential where
```
Comment (limemloh) [inline haskell-src/Concordium/ID/Types.hs:1061]: I don't see why this change is relevant for the transaction logging, but I see no problem in it

**Review by limemloh — APPROVED** [review]

**Review by td202 — APPROVED** [review]

File: haskell-src/Concordium/ID/Types.hs (hunk 1 above)
Comment (abizjak) [reply haskell-src/Concordium/ID/Types.hs:1061]: It is not relevant.

But I needed it when I was updating the dependency in concordium-client (so I could update it in the wallet-proxy).

It is a leftover from the "improve startup speed" PR and since it's so trivial I did not want to make a separate PR with it.
